docs/docs/reference/metaprogramming/staging.md

```diff
@@ -66,9 +66,9 @@ On the other hand `withQuotes` provides a `Quotes` without evaluating the expres
 ```scala
 package scala.quoted.staging
 
-def run[T](expr: Quotes ?=> Expr[T])(using toolbox: Toolbox): T = ...
+def run[T](expr: Quotes ?=> Expr[T])(using JITCompiler): T = ...
 
-def withQuotes[T](thunk: Quotes ?=> T)(using toolbox: Toolbox): T = ...
+def withQuotes[T](thunk: Quotes ?=> T)(using JITCompiler): T = ...
```

Comment:
I think having "compiler" in the name is an improvement. The usage of `JIT` might cause confusion, given that most programmers already have a stereotypical understanding of it.